src/mscorlib/shared/Interop/Windows/Kernel32/Interop.GetFullPathNameW.cs

```diff
@@ -23,7 +23,7 @@ internal static unsafe uint GetFullPathNameW(string path, uint numBufferChars, r
                 return GetFullPathNameW(path, numBufferChars, pBuffer, mustBeZero);
         }
 #else
-        internal static extern uint GetFullPathNameW(string path, uint numBufferChars, ref char buffer, IntPtr mustBeZero);
```

Comment:
Could you please update the PROJECTN workaround above as well?

Reply:
Sure, fixing